include/private/vkd3d_atomic.h

```diff
@@ -156,7 +157,8 @@ FORCEINLINE uint64_t vkd3d_atomic_uint64_decrement(uint64_t *target, vkd3d_memor
     return result;
 }
 
-FORCEINLINE uint64_t vkd3d_atomic_uint64_compare_exchange(uint64_t* target, uint64_t expected, uint64_t desired, vkd3d_memory_order order)
+FORCEINLINE uint64_t vkd3d_atomic_uint64_compare_exchange(uint64_t* target, uint64_t expected, uint64_t desired,
```

Comment:
static?

Reply:
None of the MSVC-specific functions are `static`. Not sure how useful it is to change it only for these now when it works without it anyway.

Reply:
FORCEINLINE on Windows is already static.